server/tests/src/test/java/org/infinispan/server/persistence/PersistenceIT.java

```diff
@@ -21,16 +39,64 @@
 })
 public class PersistenceIT {
 
-   public static final DatabaseServerListener DATABASE = new DatabaseServerListener("h2", "mysql", "postgres");
+   private static final String DATABASES = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_TYPES);
+   private static final String DATABASE_LIBS = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_LIBS);
+   private static final Integer ARTIFACT_ID_POSITION = 1;
```

Comment:
Only used once. I'm not sure this needs to be a global var.

Reply:
@ryanemerson this is just to avoid magic numbers. It's to have a name which explain what number does. You can lost points from SonarQube if you only put `1` as a magic number.
https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
In this case you're only using the var once and it's simply to retrieve the second argument of an array. So I would argue that doing `[1]` in place is actually much less confusing than having the var. The code `.split(":")[1]` is pretty self explanatory on it's own.